src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -166,20 +166,27 @@ def _remove_host(self, host):
 
     @orchestrator._cli_read_command(
         'orchestrator host ls',
-        desc='List hosts')
-    def _get_hosts(self):
+        'name=format,type=CephChoices,strings=json|plain,req=false',
```

Comment:
It seems like this should be implicit on every command, the way it is for the regular mon CLI commands?

Reply:
afaik it is already.